app.py

```diff
@@ -124,47 +124,31 @@ def launch_publish(self, entity_type, entity_ids):
                 if not v.get("published_files"):
                     self.log_error("Sorry, this can only be used on Versions with an associated Published File.")
                     return
-                publish_id = v["published_files"][0]["id"]
+                if len(v["published_files"]) == 1:
```

Comment:
I think we should always call the hook and let it decide what should be done. No problem to keep this logic if it makes sense, but it should be in the hook so people can change it if needed.